client/src/features/rootV2/LazyRootV2.tsx

```diff
@@ -21,10 +21,12 @@ import { Suspense, lazy } from "react";
 import PageLoader from "../../components/PageLoader";
 
 const RootV2 = lazy(() => import("./RootV2"));
+const StylesV2 = lazy(() => import("./StylesV2"));
 
 export default function LazyRootV2() {
   return (
     <Suspense fallback={<PageLoader />}>
+      {StylesV2 && <StylesV2 />}
       <RootV2 />
     </Suspense>
   );
```

Comment:
What about just importing? Something like this.
```suggestion
export default function LazyRootV2() {
  useEffect(() => {
    import("../../styles/indexV2.scss");
  }, []);
  return (
    <Suspense fallback={<PageLoader />}>
      <RootV2 />
    </Suspense>
  );
```

Reply:
Thank you @lorenzo-cavazzi . The strategy for managing styles has changed, so this code is no longer needed.